### PR TITLE
Holoparasites have a health hud display for the summoner's health

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Holoparasites can now see their summoner's health at all times. The health displayed is a percentage, with 0 being dead.
imageadd: Holoparasites now have visual flashes on the summoner's location when recalled due to range limits and when manifesting.
/:cl:

maptext memes
Adds glows when holoparasites force-recall to their summoner and when manifesting from the summoner.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
>all checks have failed

Got me all excited for nothing

**Comment by ChangelingRain** [comment]
Ain't me, it compiles fine.

File: code/_onclick/hud/hud.dm [hunk 1]
```diff
@@ -101,6 +101,8 @@ var/datum/global_hud/global_hud = new()
 	var/obj/screen/blobpwrdisplay
 	var/obj/screen/blobhealthdisplay
 
+	var/obj/screen/guardian/healthdisplay
```
Comment (neersighted) [inline code/_onclick/hud/hud.dm]: scope this to guardians I think
(ie guardian_health)

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 2]
```diff
@@ -89,6 +92,15 @@
...
 	summoner.death()
 
+/mob/living/simple_animal/hostile/guardian/proc/updatehudhealth()
+	if(summoner)
+		var/resulthealth
+		if(iscarbon(summoner))
+			resulthealth = round((summoner.health / abs(config.health_threshold_dead - summoner.maxHealth)) * 100)
+		else
+			resulthealth = round((summoner.health / summoner.maxHealth) * 100)
+		hud_used.healthdisplay.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#efeeef'>[resulthealth]%</font></div>"
```
Comment (neersighted) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: align/valign are horribad, sure you have to use them?

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 3]
```diff
@@ -133,7 +147,7 @@
 		return
 	PoolOrNew(/obj/effect/overlay/temp/guardian/phase/out, get_turf(src))
 	unbuckle_mob(force=1)
-	loc = summoner
+	forceMove(summoner)
```
Comment (neersighted) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Unrelated changes or does this include another unmerged PR?

Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: This is something I forgot to do with the previous forcemove changes, should have the same result as currently with less lighting bugs.

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: >should

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: I have no fucking idea, this is copied from blob health/power displays and I don't think there's another way to center it but I know nothing about this

**Comment by Razharas** [comment]
Aight
